Sources/XLPagerTabStrip/TwitterPagerTabStripViewController.swift

```diff
@@ -23,6 +23,35 @@
 // THE SOFTWARE.
 
 import Foundation
+import UIKit
+#if SWIFT_PACKAGE
+import FXPageControl
+#endif
+
+extension UIWindow {
```

Comment:
I think this extension should be moved to a separate file named something like UIWindow+Pager.swift or so